doc/src/Glossary.dox

```diff
@@ -116,10 +116,10 @@
  *
  * The following are examples of perfectly valid entity references:
  *
- *  @li <tt>myasset://DVR/ep01/build/prop/table?v=6&t=model&f=alembic</tt>
- *  @li <tt>e1d2e41c-9594-11e2-9619-f23c91ae2917</tt>
- *  @li <tt>{'job':'DVR', 'path':['ep01','build','prop','table'],
- *  'version':'6', 'type':'model', 'format':'alembic'}</tt>
+ *  @li `myasset://DVR/ep01/build/prop/table?v=6&t=model&f=alembic`
```

Comment:
[fix] For some reason this file doesn't process the backticks

Reply:
Fixed the file, but the last ref still didn't work with backticks so reverted [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/94/commits/01866e4f6b3b35fe28072ccc6386a01e4c323e68).